crates/gen/src/windows.rs

```diff
@@ -1,11 +1,25 @@
 /// Returns the build's `.windows` directory in the root of the workspace as a `PathBuf`.
 pub fn build_windows_dir() -> std::path::PathBuf {
-    let mut path = workspace_dir();
+    build_windows_dir_at_workspace(&workspace_dir())
+}
+
+/// Returns the build's `.windows` directory in the passed workspace directory as a `PathBuf`.
+pub fn build_windows_dir_at_workspace(workspace_dir: &std::path::PathBuf) -> std::path::PathBuf {
```

Comment:
`build_windows_dir_at_workspace` => `workspace_windows_dir`

Reply:
`workspace_windows_dir` sounds good.

Reply:
done - renamed it in last update